app/email_utils.py

```diff
@@ -1457,3 +1462,44 @@ def get_dmarc_status(msg: Message) -> Optional[DmarcCheckResult]:
             return dmarc_result
 
     return None
+
+
+def generate_verp_email(verp_type: VerpType, tx_id: int) -> str:
+    data = [verp_type.bounce.value, tx_id, int(time.time())]
+    json_payload = json.dumps(data).encode("utf-8")
+    # Signing without itsdangereous because it uses base64 that includes +/= symbols and lower and upper case letters.
+    # We need to encode in base32
+    signed_payload = hmac.new(
+        FLASK_SECRET.encode("utf-8"), json_payload, "sha224"
```

Comment:
I didn't want to break existing intstallations but I'll add an optional config, otherwise use the flask secret. This way we won't break other user installations.

Reply:
Just to be sure. The key should be at least 56 chars long or it starts to lose 'effectiveness'. If it's shorter than that, would you raise an exception?

Reply:
I've added an exception if the new token is too short. By default I get the flask one and add a fixed string.